tests/datasource/fluent/test_pandas_filesystem_datasource.py

```diff
@@ -637,7 +637,7 @@ def test_pandas_sorter(
         order_by=order_by,
     )
     batches = asset.get_batch_list_from_batch_request(asset.build_batch_request())
-    assert (len(batches)) == 36
+    assert len(batches) == 37
 
```

Comment:
Added the new CSV and added a table to the existing SQLite DB with the same information